CRM/Utils/System/WordPress.php

```diff
@@ -1283,6 +1283,15 @@ public function sessionStart() {
       session_start();
     }
   }
+  
+  /**
```

Comment:
This seems intuitively sensible.

@bastienho I think the test bot is [complaining](https://test.civicrm.org/job/CiviCRM-Core-PR/51858/checkstyle/) because the blank line (1286 above) has some invisible space characters. If you can delete the space-characters, then it should accept it.